templates/registration/login.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
```

Comment:
The template correctly checks for form errors and displays an error message if the credentials are invalid. This provides feedback to the user when login fails.